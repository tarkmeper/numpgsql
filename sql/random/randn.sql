SELECT random_seed(1);

SELECT random_randn(2);

SELECT random_randn(2,2);




