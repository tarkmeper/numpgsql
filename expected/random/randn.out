SELECT random_seed(1);
 random_seed 
-------------
 
(1 row)

SELECT random_randn(2);
             random_randn              
---------------------------------------
 {-0.54974617895545,-1.40287270912792}
(1 row)

SELECT random_randn(2,2);
                               random_randn                                
---------------------------------------------------------------------------
 {{1.58275229197514,-1.04514681044202},{0.25759354129149,-1.959394569747}}
(1 row)

